Makefile

```diff
@@ -239,7 +239,6 @@ clean:
 	rm -fr include
 	rm -fr ./SparseGrids/*.o
 	rm -fr ./DREAM/*.o
-	rm -fr ./DREAM/Optimization/*.o
```

Comment:
Why deleting this? It was needed for cleanup after the compilation process, the object files are in `./DREAM/Optimization/*.o`

Reply:
This was a mistake. I was playing around with Makefile and forgot to put this line back. It should be correct in the latest push.